scripts/ddp-RxTerms-script/src/index.ts

```diff
@@ -0,0 +1,53 @@
+import * as XLSX from "xlsx";
+import { RxNormDrugOption } from "./models";
+import { parseToHOCON, writeFile } from "./utils/common";
+
+const inputFileName = process.argv[2] || "RxTerms202202.txt";
+const outputFileName = process.argv[3] || "output.txt";
+
+const translations: string[] =  process.argv.filter(arg => ['en', 'es', 'ru'].includes(arg));
+
+const workBook = XLSX.readFile(inputFileName);
+
+const workSheet = workBook.Sheets[workBook.SheetNames[0]];
+
+const rxNormArray = XLSX.utils.sheet_to_json(workSheet) as RxNormDrugOption[];
```

Comment:
The parsing seems to work well! The one problem I am seeing is that it converts the drug ids to numbers because they all look like integers. But we need strings for stable ids. Suggested way to handle it below in the suggest code.